playbooks/group_vars/webportals_pro.yml

```diff
@@ -0,0 +1,5 @@
+# Config override for pro servers (hosts in webportals_pro group)
```

Comment:
Didn't we agreed that we will not introduce any more group vars files and will use hosts.ini instead?